mev-build-rs/src/reth_builder/payload_builder.rs

```diff
@@ -63,14 +63,14 @@ fn assemble_txs_from_pool<Pool: reth_transaction_pool::TransactionPool>(
     let effective_gas_limit = block_gas_limit - context.build.gas_reserve;
     while let Some(pool_tx) = best_txs.next() {
         if context.is_cancelled() {
-            return Ok(())
+            return Ok(());
```

Comment:
I think this is from using stable rustfmt, when this repo has a config file using nightly fmt